src/DynamoDbModel.php

```diff
@@ -195,10 +218,10 @@ public static function all($columns = [], $limit = -1)
         return $model->getAll($columns, $limit);
     }
 
-    public static function first($columns = [])
+    public function first($columns = [])
```

Comment:
no, you're absolutely right. I need to update the tests obviously. If we can find a solution that can work with static methods too, that would be great. I'll look more into it this weekend. Thanks!